### PR TITLE
Added link to tutorials

### PR DESCRIPTION
**Pull request by shaunsmith**
Pointed README tutorial link to tutorial project and renamed old tutorial link to 'quickstart'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,8 @@ you can update your code and run `fn deploy myapp` again.
 
 ## To Learn More
 
-* Visit our Functions [Tutorial Series](examples/tutorial/)
+* Try out a [Tutorial](../tutorials/README.md) 
+* Visit our Functions [Quickstart Series](examples/tutorial/)
 * See our [full documentation](docs/README.md)
```
Comment (denismakogon) [inline README.md]: worth adding links to any videos we have, no?

File: README.md [hunk 2]
```diff
@@ -115,7 +115,8 @@ you can update your code and run `fn deploy myapp` again.
 
 ## To Learn More
 
-* Visit our Functions [Tutorial Series](examples/tutorial/)
+* Try out a [Tutorial](../tutorials/README.md) 
+* Visit our Functions [Quickstart Series](examples/tutorial/)
```
Comment (treeder) [inline README.md]: Quickstart is already in the main README, should call this something else. Language hello world examples perhaps?

Comment (denismakogon) [reply README.md]: Maybe `Functions development quickstart series`?

File: README.md (hunk 1 above)
Comment (denismakogon) [reply README.md]: @shaunsmith could you also add youtube channel link?

**Comment by rdallman** [comment]
@shaunsmith re-open when you sign the cla! (I kid) -- this seems stale, re-open when updated please.

**Pull request closed without merging**